### PR TITLE
Change config to use tailwind

### PR DESCRIPTION
**Pull request by spiess-demos**
**Ticket:** https://yaits.demos-deutschland.de/T27594

This PR includes tailwind within demosplan. 

To test it, just run `yarn install`, followed by a `yarn dev:<project>`.

| build | time |
| --- | --- |
| no postcss-loader | 16s |
| with postcss-loader <- this was the status quo | 19s |
| no postcss-loader, with demosplan-ui | 26s |
| with postcss-loader, with demosplan-ui | 43s |
| with postcss-loader, with demosplan-ui, skip 3rd party css | 30s |
| **with postcss-loader, with demosplan-ui, skip 3rd party css, add sass-embedded** | **21s** |

So, at the moment this adds 2 seconds to the build.

### Todos

- [x] Update demosplan-ui to include the below PR

### Linked PRs

- https://github.com/demos-europe/demosplan-ui/pull/294

### CODE REVIEW

File: client/fe/webpack/moduleRules.js [hunk 1]
```diff
@@ -127,24 +120,19 @@ const moduleRules =
     {
       test: /\.s?css$/,
       use: [
+        MiniCssExtractPlugin.loader,
         {
-          loader: 'vue-style-loader'
```
Comment (spiess-demos) [inline client/fe/webpack/moduleRules.js:131]: booh

File: client/fe/config/config.js [hunk 2]
```diff
@@ -75,22 +75,23 @@ class Config {
           /ol-.+/,
           /plyr-.+/,
           /uppy-.+/,
-          /color-.+/,
+          /^color-.+/,
```
Comment (salisdemos) [inline client/fe/config/config.js:78]: why that? Or other way round: shouldn't we add the `^` to (almost) all the entries?

Comment (spiess-demos) [reply client/fe/config/config.js:78]: > shouldn't we add the ^ to (almost) all the entries?

This is probablyright but i did not want to risk breaking stuff, and i did not want to change it everywhere without cheching side effects... which would take some time for those big ones.

**Review by salisdemos — DISMISSED** [review]

**Review by salisdemos — DISMISSED** [review]

**Comment by spiess-demos** [comment]
> Just a side note, I think we should do demosplan-ui updates more explicitly (in a separate PR)

but in this case, the changes made here depended on the new version.

**Pull request merged**